src/app/services/baw-api/baw-api.service.ts

```diff
@@ -258,22 +266,29 @@ export abstract class BawApiService<T extends AbstractModel> {
  * Filter paging metadata from api response
  */
 export interface Paging {
+  /** Current page number */
   page?: number;
+  /** Maximum number of items per page */
   items?: number;
+  /** Total number of items for filter request */
   total?: number;
+  /** Maximum page number */
   maxPage?: number;
+  /** @deprecated */
   current?: string;
+  /** @deprecated */
   previous?: string;
+  /** @deprecated */
```

Comment:
why deprecated?

Reply:
AFAIK there is no use for these values in the new workbench-client. They don't provide much useful information that the system can't already calculate, and realistically I can't think of a single scenario these would be the best option to use. I set to deprecated to hint to future developers that there is likely a better solution for tracking what the current, previous and next filters are.

Reply:
See HATEOAS

Ideally our API returns URLs to most of its own resources. It doesn't do that for most... though that's a separate issue.

The client is not the only client for the API. Other clients may not have enough information to form their own links. So this feature certainly is deprecated.

Maybe a comment explaining they're just not used. Or even, remove them from the type system altogether

Reply:
I think I'll just remove them from the type system